docs/hardware/devices/linux-native-hardware/linux-native-hardware.mdx

```diff
@@ -114,13 +136,120 @@ dtoverlay=uart0
 
 - The correct port for UART GPS on the Pi 5 after a reboot is `/dev/ttyAMA0`.
 - The correct port for UART GPS on earlier Pi versions after a reboot is `/dev/ttyS0`.
+- You may need to disable the serial console on a Pi and then reboot
+```shell
+sudo raspi-config nonint do_serial_cons 1 # Disable Serial Console
+```
 
 ### Persistence
 
 - The persistent .proto db files of the portduino version of meshtasticd are stored under: `/root/.portduino/default/prefs/`.
 
 ### Advanced Setup and Troubleshooting
 
+#### Creating and Enabling a Systemctl Script (non .deb installs)
+
+To Setup the device to start and stop meshtasticd as as service using systemctl you can setup the service unit using the instructions below.
+
+Create the service unit file:
+
+Create a new file in the /etc/systemd/system/ directory with a name like meshtasticd.service.
+```shell
+sudo nano /etc/systemd/system/meshtasticd.service
+```
+Add the following content to the file:
+
+```plaintext
+[Unit]
+Description=Meshtastic Daemon
+After=network.target
+
+[Service]
+ExecStart=/usr/sbin/meshtasticd
+Restart=always
+User=root
+Group=root
+StandardOutput=file:/var/log/meshtasticd.log
```

Comment:
For now, let's drop the txt log output and log rotation. I'm very tempted to add this to the official .deb, but until that happens, let's keep the instructions in sync.

Reply:
Makes sense, removed StandardOutput= entry from service file.